app/models/auth_source.rb

```diff
@@ -26,8 +26,8 @@ class AuthSource < ActiveRecord::Base
 
   validates :name, :presence => true, :uniqueness => true, :length => { :maximum => 60 }
 
-  scope :non_internal, lambda { where("type NOT IN (?)", ['AuthSourceInternal', 'AuthSourceHidden']) }
-  scope :except_hidden, lambda { where('type <> ?', 'AuthSourceHidden') }
+  scope :non_internal,  -> { where("type NOT IN (?)", ['AuthSourceInternal', 'AuthSourceHidden']) }
```

Comment:
Please avoid lining everything up too much, otherwise everybody needs to reindent any time it's changed or added to.